pkg/alexa/l10n/l10n.go

```diff
@@ -170,7 +173,8 @@ func (l *Locale) Set(key string, values []string) {
 }
 
 func (l *Locale) Get(key string, args ...interface{}) string {
-	return l.TextSnippets.GetFirst(key, args...)
+	t, _ := l.TextSnippets.GetFirst(key, args...)
```

Comment:
You are ignoring an error.

Reply:
now allowing the "user" to `GetErrors()` if he wishes to know which keys are used, but not translated.